src/main/scala/org/broadinstitute/dsde/workbench/sam/api/ResourceRoutes.scala

```diff
@@ -370,4 +376,13 @@ trait ResourceRoutes extends UserInfoDirectives with SecurityDirectives with Sam
         complete(resourceService.listResourceChildren(resource, samRequestContext).map(children => StatusCodes.OK -> children))
       }
     }
+
+  def deletePolicy(policyId: FullyQualifiedPolicyId, userInfo: UserInfo, samRequestContext: SamRequestContext): server.Route =
+    delete {
+      requireAction(policyId.resource, SamResourceActions.alterPolicies, userInfo.userId, samRequestContext) {
+        requireAction(policyId.resource, SamResourceActions.deletePolicy(policyId.accessPolicyName), userInfo.userId, samRequestContext) {
```

Comment:
I think this should be `requireOneOfAction`. Both actions are not required, only 1.

Reply:
Just to clarify, the wording in the ticket/design doc is 

> 403 if user has access to resource but is missing one of
> alter_policies
> delete_policy::{policyName}

which I interpreted as both being required. However, now that you mention it, it definitely makes sense to me that if you have either of these actions you'd be able to delete a policy. So is it simply an error in the ticket?